imdb-keras/README.md

```diff
@@ -0,0 +1,73 @@
+# IMDB example - Keras version
+This classic example of sentiment analysis is well suited both as a lightweight test when learning FEDn and developing on FEDn in psedo-distributed mode. A normal high-end laptop or a workstation should be able to sustain at least 5 clients. The example is also useful for general scalability tests in fully distributed mode. 
+
+### Local training and test data
+To download and prepare the partitioned dataset:
+``` bash
+python create_data_partitions.py
+```
+
+## Configuring the Reducer  
+Navigate to 'https://localhost:8090' (or the url of your Reducer) and follow instructions to upload the compute package in 'package/package.tar.gz' and the initial model in 'initial_model/initial_model.npz'. 
+
+## Attaching a client to the federation
+
+1. First, download 'client.yaml' from the Reducer 'Network' page, and replace the content in your local 'client.yaml'. 
+2. Start a client. Here there are different options (see below): 
+    - Docker 
+    - docker-compose
+    - [Native client (OSX/Linux)](https://github.com/scaleoutsystems/examples/tree/main/how-tos/start-native-fedn-client)
+
+#### Docker
+1. Build the image
+
+``` bash
+docker build . -t imdb-client:latest
+```
+
+2. Start a client (edit the path of the volume mounts to provide the absolute path to your local folder.)
+```
+docker run -v /absolute-path-to-this-folder/data/:/app/data:ro -v /absolute-path-to-this-folder/client.yaml:/app/client.yaml --network fedn_default imdb-client fedn run client -in client.yaml 
```

Comment:
The path will not be correct after the data partitions are generated, right? ../data/clients/0 ?